pkg/internal/assert/assert_test.go

```diff
@@ -64,6 +64,34 @@ func TestExpectError(t *testing.T) {
 	})
 }
 
+func TestBoolEqual(t *testing.T) {
+	t.Parallel()
+	t.Run("not equal", func(t *testing.T) {
+		t.Parallel()
+		var f fakeT
+		BoolEqual(&f, true, false)
+		if int(f) == 0 {
+			t.Fatalf("Expected t.Errorf to be called but it was not")
+		}
+		BoolEqual(&f, false, true)
```

Comment:
second test case with the swapped arguments can be omitted?

Reply:
Thanks, done.